src/Settings.cpp

```diff
@@ -155,3 +176,23 @@ bool Settings::getStartInTray() const
 
     return enable;
 }
+
+void Settings::setAutoStart()
+{
+    auto dfPath = getDesktopFilePath();
+    if(dfPath.empty())
+    {
+        std::cerr << "setAutoStart() failed. Couldn't find .desktop file" << std::endl;
```

Comment:
Please start with the class name in the print message i.e.  `Settings: Failed to find desktop file`.